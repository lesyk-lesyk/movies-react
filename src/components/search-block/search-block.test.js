import React from 'react';
import { SearchBlock } from './index';
import renderer from 'react-test-renderer';

jest.mock('../site-name/site-name');
jest.mock('../search-field/search-field');
jest.mock('../search-controls/search-controls');

describe('SearchBlock', () => {
  it('renders', () => {
    const tree = renderer
      .create(<SearchBlock />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
