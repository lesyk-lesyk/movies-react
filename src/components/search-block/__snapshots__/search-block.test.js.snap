// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBlock renders 1`] = `
<div
  className="container search-container"
>
  <SiteName />
  <h1
    className="search-header"
  >
    Find your movie
  </h1>
  <SearchField />
  <SearchControls />
</div>
`;
